### PR TITLE
[tests-only][full-ci] add tests to list space files shared with group when auto sync is disable

### PR DESCRIPTION
**Pull request by nabim777**
## Description

In this PR following scenarios are added related to sharingNG:
```
-   Scenario: group member lists the file shared with them from project-space when auto-sync is disabled
-   Scenario: group member lists the space folder shared with them from project-space when auto-sync is disabled
```

## Related Issue

- Part of https://github.com/owncloud/ocis/issues/8318

## How Has This Been Tested?

- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [X] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by S-Panta** [comment]
LGTM

File: tests/acceptance/features/apiSharingNg/sharedWithMe.feature [hunk 1]
```diff
@@ -2974,4 +2974,444 @@ Feature: an user gets the resources shared to them
         }
       }
     }
-    """
\ No newline at end of file
+    """
+
+
+  Scenario: group member lists the file shared with them when auto-sync is disabled
```
Comment (saw-jan) [inline tests/acceptance/features/apiSharingNg/sharedWithMe.feature:2980]: This scenario is for project space so would be better to indicate that in scenario title
```suggestion
  Scenario: group member lists the file shared with them when auto-sync is disabled
```

File: tests/acceptance/features/apiSharingNg/sharedWithMe.feature [hunk 2]
```diff
@@ -2974,4 +2974,444 @@ Feature: an user gets the resources shared to them
         }
       }
     }
-    """
\ No newline at end of file
+    """
+
```
Comment (saw-jan) [inline tests/acceptance/features/apiSharingNg/sharedWithMe.feature:2978]: tests with `auto-sync` can be moved it it's on feature file. the file has already become too long

**Review by saw-jan — CHANGES_REQUESTED** [review]

File: tests/acceptance/features/apiSharingNg/sharedWithMe.feature [hunk 3]
```diff
@@ -2974,4 +2974,444 @@ Feature: an user gets the resources shared to them
...
+    And user "Carol" has been created with default attributes and without skeleton files
+    And group "grp1" has been created
+    And the following users have been added to the following groups
+      | username | groupname |
+      | Brian    | grp1      |
+      | Carol    | grp1      |
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "projectSpace" with the default quota using the Graph API
+    And user "Alice" has created a folder "folderToShare" in space "projectSpace"
+    And user "Brian" has disabled the auto-sync share
```
Comment (amrita-shrestha) [inline tests/acceptance/features/apiSharingNg/sharedWithMe.feature:3226]: if brain set auto-sync disable in all scenario then we can shift this step to background

File: tests/acceptance/features/apiSharingNg/sharedWithMeSyncDisabled.feature [hunk 4]
```diff
@@ -1493,3 +1488,441 @@ Feature: listing sharedWithMe when auto-sync is disabled
         }
       }
       """
+
+
+  Scenario: group member lists the file shared with them from project-space when auto-sync is disabled
+    Given using spaces DAV path
+    And user "Carol" has been created with default attributes and without skeleton files
```
Comment (SagarGi) [inline tests/acceptance/features/apiSharingNg/sharedWithMeSyncDisabled.feature:1495]: is carol even needed for this scenario?

File: tests/acceptance/features/apiSharingNg/sharedWithMeSyncDisabled.feature [hunk 5]
```diff
@@ -1493,3 +1488,441 @@ Feature: listing sharedWithMe when auto-sync is disabled
...
+    And group "grp1" has been created
+    And the following users have been added to the following groups
+      | username | groupname |
+      | Brian    | grp1      |
+      | Carol    | grp1      |
```
Comment (SagarGi) [inline tests/acceptance/features/apiSharingNg/sharedWithMeSyncDisabled.feature:1500]:
```suggestion
```

File: tests/acceptance/features/apiSharingNg/sharedWithMeSyncDisabled.feature (hunk 4 above)
Comment (nabim777) [reply tests/acceptance/features/apiSharingNg/sharedWithMeSyncDisabled.feature:1493]: Yes its not needed

**Review by saw-jan — APPROVED** [review]
LGTM :+1:

**Review by SagarGi — APPROVED** [review]
LGTM :+1:

**Pull request merged**